opentelemetry-api/src/opentelemetry/context/propagation/tracestatehttptextformat.py

```diff
@@ -0,0 +1,20 @@
+import opentelemetry.trace as trace
+from opentelemetry.context.propagation import httptextformat
+
+
+class TraceStateHTTPTextFormat(httptextformat.HTTPTextFormat):
```

Comment:
I figured since we want to propagate TraceState by default, we can eliminate an invalid propagator by sticking this here for now.

Reply:
Personally I'd prefer to keep the API doing nothing. But open-telemetry/opentelemetry-specification#208 and open-telemetry/opentelemetry-specification#183 are still open, so I'm OK with this for now.

Reply:
After reading the rest of the PR, I see that this might just be a naming issue. Would you expect `SpanContextHTTPTextFormat` to describe the same thing?

Reply:
I think the proper name would be `W3CTraceContextHTTPTextFormatter` (or some abbreviation thereof).

Reply:
@c24t @toumorokoshi 
Just to clarify, the intent behind `TraceStateHTTPTextFormat` is to actually cover `SpanContext` right? So it should simply just be renamed?

Reply:
As I understand it, yes.